docs/book/src/quinn/data-transfer.md

```diff
@@ -0,0 +1,165 @@
+# Data Transfer
+
+In the [previous chapter](set-up-connection.md) we characterized how to set up an [Endpoint][Endpoint]
+and then get access to a [NewConnection][NewConnection] instance.
+Now we will continue with the subject of sending data over this connection.
+
+## Multiplexing
+
+Multiplexing is the act of combining data from multiple streams into a single stream. 
+This can have a significant positive effect on the performance of the application. 
+QUIC it itself does not perform multiplexing instead it gives the programmer full control over this process. 
```

Comment:
This sentence doesn't make sense to me. QUIC does multiplex frames for all streams into what is conceptually a single stream of QUIC packets.

Reply:
I mean, like in HTTP/2 multiplexing is performed for you, in QUIC you're responsible for it.

Reply:
I don't agree. In the Quinn API, the user is certainly not on the hook to merge together parts of different streams into particular packets.

Reply:
That's right. Maybe I need to rephrase it to the user being in control over how to allocate streams and spread data over them.